.circleci/config.yml

```diff
@@ -47,6 +48,19 @@ references:
       name: Coverage
       command: bundle exec undercover
 
+  setup_code_climate: &setup_code_climate
```

Comment:
These references (`setup_code_climate` and `run_coverage_codeclimate`) are not used